src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java

```diff
@@ -276,6 +279,21 @@ private void handleDirectCommand(String channel, String sender, String login, St
         }
     }
 
+    private void sendFallbackMessage(String channel, String payload) {
+        if(StringUtils.containsIgnoreCase(payload, "thank")) {
+            sendMessage(channel, "You're welcome");
+        }
+        if(StringUtils.startsWithIgnoreCase(payload, "hello")) {
+            sendMessage(channel, "Hello you");
```

Comment:
done. good idea indeed. As for the bot infinite answering, hopefully we should be fine if nobody does it on purpose since the currently expectable cases of some bots sending messages are already ignored before reaching that code portion.